controllers/storagecluster/placement.go

```diff
@@ -20,6 +20,12 @@ func getPlacement(sc *ocsv1.StorageCluster, component string) rookCephv1.Placeme
 	} else {
 		in := defaults.DaemonPlacements[component]
 		(&in).DeepCopyInto(&placement)
+		// default value for mds pod anti affinity is set to the filesystem name
+		// label rook_file_system is added to the mds pod using rook operator
+		// should be set before appendNodeRequirements() so to preserve the default values
```

Comment:
```suggestion
```

No need of this line as in the `appendNodeRequirements` we don't do anything with the `PodAntiAffinity`